src/TabPane.js

```diff
@@ -0,0 +1,22 @@
+
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  tabId: PropTypes.any
+};
+const contextTypes = {
+  activeTabId: PropTypes.any
+};
+
+export default function TabPane({ className, tabId, children }, context) {
```

Comment:
could you move these props into something like the pattern in other components, [example](https://github.com/reactstrap/reactstrap/blob/af874bc9001be3bf692a998714ebe7aa4f3d7bbb/src/Card.js#L18-L26).

This makes it easier to pass through attributes/props that aren't explicitly called out.